CMakeLists.txt

```diff
@@ -132,9 +132,22 @@ endif()
 option(PSMOVE_USE_LOCAL_OPENCV "Use locally-built OpenCV (static linking)" OFF)
 if (PSMOVE_USE_LOCAL_OPENCV)
     # Force tracker build with a locally-built opencv library installation
-    set(OPENCV_SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR}/opencv)
-    link_directories(${OPENCV_SRC_DIR}/build/lib)
-    link_directories(${OPENCV_SRC_DIR}/build/3rdparty/lib)
+    
+    # allow for a custom OPENCV_SRC_DIR to be passed
+    if (NOT DEFINED OPENCV_SRC_DIR)
```

Comment:
Can you expose this variable so that an interactive `ccmake`/`cmake-gui` shows these options?

http://stackoverflow.com/questions/8709877/cmake-string-options

Should be enough to add something like this to the top of the file (untested):

```
set(OPENCV_SRC_DIR "${CMAKE_CURRENT_SOURCE_DIR}/opencv" CACHE STRING "Directory where the OpenCV sources are")
```